docs/OV_Runtime_UG/deployment/deployment-manager-tool.md

```diff
@@ -69,23 +69,23 @@ The target device selection dialog is displayed:
+Use the options provided on the screen to complete selection of the target devices and press **Enter** to proceed to the package generation dialog. To interrupt the generation process and exit the program, type **q** and press **Enter**.
 
-Once you accept the selection, the package generation dialog is displayed:
+Once the selection is accepted, the package generation dialog will appear:
   
 ![Deployment Manager configuration dialog](../img/configuration_dialog.png)
 
-The target devices you have selected at the previous step appear on the screen. To go back and change the selection, type **b** and press **Enter**. Use the options provided to configure the generation process, or use the default settings.
+The target devices selected in the previous step appear on the screen. To go back and change the selection, type **b** and press **Enter**. Use the options provided to configure the generation process, or use the default settings:
    
-* `o. Change output directory` (optional): Path to the output directory. By default, it's set to your home directory.
+* The `o. Change output directory` (optional) - a path to the output directory. By default, it is set to your home directory.
 
-* `u. Provide (or change) path to folder with user data` (optional): Path to a directory with user data (IRs, models, datasets, etc.) files and subdirectories required for inference, which will be added to the deployment archive. By default, it's set to `None`, which means you will separately copy the user data to the target system.
+* The `u. Provide (or change) path to folder with user data` (optional) - a path to a directory with user data (OpenVINO IRs, models, datasets, etc.) files and subdirectories required for inference, which will be added to the deployment archive. By default, it is set to `None`, which means that copying the user data to the target system will be performed separately.
 
-* `t. Change archive name` (optional): Deployment archive name without extension. By default, it is set to `openvino_deployment_package`.
+* The `t. Change archive name` (optional) - a deployment archive name without extension. By default, it is set to the `openvino_deployment_package`.
  
-Once all the parameters are set, type **g** and press **Enter** to generate the package for the selected target devices. To interrupt the generation process and exit the program, type **q** and press **Enter**.
+Once all the parameters have been set, type **g** and press **Enter** to generate the package for the selected target devices. To interrupt the generation process and exit the program, type **q** and press **Enter**.
 
-The script successfully completes and the deployment package is generated in the specified output directory. 
+Once the script has successfully completed, the deployment package is generated in the specified output directory. 
 
 @sphinxdirective
 
```

Comment:
Need to update the heading of "Run Standard CLI Mode".

Reply:
It's been updated ;)